lib/block-supports/duotone.php

```diff
@@ -300,9 +301,9 @@ function gutenberg_render_duotone_filter_preset( $preset ) {
 					"
 				/>
 				<feComponentTransfer color-interpolation-filters="sRGB" >
-					<feFuncR type="table" tableValues="<?php echo esc_attr( implode( ' ', $duotone_values['r'] ) ); ?>" />
-					<feFuncG type="table" tableValues="<?php echo esc_attr( implode( ' ', $duotone_values['g'] ) ); ?>" />
-					<feFuncB type="table" tableValues="<?php echo esc_attr( implode( ' ', $duotone_values['b'] ) ); ?>" />
+					<feFuncR type="<?php echo $duotone_type; ?>" tableValues="<?php echo esc_attr( implode( ' ', $duotone_values['r'] ) ); ?>" />
```

Comment:
Is there a reason why the type is not escaped with esc_attr?

Reply:
Done in db3fc4a